scripts/add-list-items.js

```diff
@@ -0,0 +1,17 @@
+export default function addListItem(event, input, list) {
```

Comment:
this feels specific to the to-do.js implementation. I would move the file to the same folder (blocks/todo)